Makefile

```diff
@@ -0,0 +1,200 @@
+MODULE   = $(shell env GO111MODULE=on $(GO) list -m)
```

Comment:
What can I say, I like consistent `Makefile`s across projects I'm working on. =) This, like a number of other things I've called out, will go away once we merge Resolution into Pipeline.